### ISSUE TITLE
R-Kernel: No output of cell that runs python(r-keras)

### ISSUE THREAD

**Issue by reisman234** [jupyterlab/jupyterlab#5868]
Hi!

I'm not quite sure if this problem is more IRkernel related or jupyterlab. I work on a notebook in R in combination with r-keras, to run deep neural networks. For this I have created a docker image that runs jupyterlab.

**The bug**
My Problem with jupyterlab is that i dont get any std-output for cells that runs specific r-keras code, for example when i start a model training. I found out that I get the missing output in the terminal which has started the docker container.
This behaviour is in JupyterLab and also in Jupyter Notebook

***Missing Output***
![grafik](https://user-images.githubusercontent.com/44891585/51191727-4ecac600-18e5-11e9-9218-cb4516c97c92.png)

***Terminal Output***
![grafik](https://user-images.githubusercontent.com/44891585/51191913-b3862080-18e5-11e9-9e4b-f25b990d95ad.png)

**To Reproduce**
The docker image that i have created, is based on the r-notebook Dockerfile form docker-stacks. My Changes was:
- change username
- change the base image to nvidia/cuda:9.0-cudnn7-runtime-ubuntu16.04 to have gpu access
- install keras and tensorflow via pip
- add r-keras packes in installation. 

**Environment**

- OS: Ubuntu 16.04
- Browser chrome 71 firefox 64
- JupyterLab 0.35.4
- R 3.4.1
- Output sessioninfo()
```
R version 3.4.1 (2017-06-30)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.5 LTS

Matrix products: default
BLAS: /opt/conda/lib/R/lib/libRblas.so
LAPACK: /opt/conda/lib/R/lib/libRlapack.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] keras_2.1.6

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.0      lattice_0.20-35 zeallot_0.1.0   digest_0.6.18  
 [5] crayon_1.3.4    IRdisplay_0.6.2 grid_3.4.1      R6_2.2.2       
 [9] repr_0.17       jsonlite_1.5    magrittr_1.5    evaluate_0.11  
[13] tfruns_1.4      stringi_1.2.4   uuid_0.1-2      whisker_0.3-2  
[17] Matrix_1.2-14   reticulate_1.10 IRkernel_0.8.14 tools_3.4.1    
[21] stringr_1.3.1   compiler_3.4.1  base64enc_0.1-3 tensorflow_1.9 
[25] htmltools_0.3.6 pbdZMQ_0.3-2   
```

**Test-Notebook**
[Test.zip](https://github.com/jupyterlab/jupyterlab/files/2760400/Test.zip)

Best reagrds
reisman

**Comment by saulshanabrook** [comment]
Hey @reisman234 thanks for leaving such detailed notes. 

> This behaviour is in JupyterLab and also in Jupyter Notebook

Since this behavior exists in both versions, it makes it less likely it will be fixed in this repo. 

Do you know which R kernel you are using? It might make sense to open the issue there or in the r keras repo.

**Comment by reisman234** [comment]
Hi @saulshanabrook and thanks for your response. 

> Do you know which R kernel you are using? 

The installed R-kernel is `IRkernel_0.8.14`, because I saw that a newer Version in the IRKernel repo exits (0.8.15) i created an second Dockerfile in which i install a plain R-Environment (not via conda). With that i could now install cran-packages directly or via Devtools from github. So I checked also the newest Version from GitHub but it made no changes about my problem. 

I will open a new issue in the IRkernel repo.
